setup.py

```diff
@@ -20,4 +20,9 @@
     ],
     include_package_data=True,
     zip_safe=False,
```

Comment:
Quick Question: should I keep this as false or true? the package is pretty small.

Reply:
I think you can set it to true